graylog2-web-interface/src/components/users/TimeoutInput.tsx

```diff
@@ -81,6 +80,8 @@ const TimeoutInput = ({ value: propsValue, onChange }: Props) => {
     setUnit(newUnit);
   };
 
+  // @ts-ignore
+  // @ts-ignore
```

Comment:
I think these slipped in through IntelliJ?

Reply:
yes :smile: